### PR TITLE
Update DG Use Cases

### LINKED ISSUE #97: Update DG Use Cases

### PR DESCRIPTION
**Pull request by XinghuaJulia**
Fixes #97

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -311,100 +311,155 @@ _{Explain here how the data archiving feature will be implemented}_
...
 
 **MSS**
 
-1.  anUdder requests to list all their contacts
-2.  FindingbrUdders shows a list of contacts
-3.  anUdder requests to delete a specific contact from the list
-4.  FindingbrUdders deletes the contact
+1.  the user <u> lists all Udders </u>.
+2. the user requests to delete a specific Udder from the list.
+3. FindingbrUdders deletes the Udder and displays the new Udder list.
```
Comment (ulricolo7) [inline docs/DeveloperGuide.md:326]: Is there supposed to be one whitespace or two?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -311,100 +311,155 @@ _{Explain here how the data archiving feature will be implemented}_
...
-**Use case: Delete anUdder**
+**Use case: Delete an Udder**
 
 **MSS**
 
-1.  anUdder requests to list all their contacts
-2.  FindingbrUdders shows a list of contacts
-3.  anUdder requests to delete a specific contact from the list
-4.  FindingbrUdders deletes the contact
+1.  the user <u> lists all Udders </u>.
```
Comment (PeterHW963) [inline docs/DeveloperGuide.md:324]: extra whitespace, can remove

**Review by PeterHW963 — COMMENTED** [review]
Mostly LGTM! Just the extra whitespace (I'm not sure if it's intentional)

**Review by ulricolo7 — COMMENTED** [review]
Just one comment! LGTM otherwise

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ulricolo7) [reply docs/DeveloperGuide.md:324]: ee duplicate comment

**Comment by ulricolo7** [comment]
Okay nice LGTM!

**Review by ulricolo7 — COMMENTED** [review]
Thanks for the changes! LGTM

**Pull request merged**